src/main/kotlin/io/zerobase/smarttracing/Main.kt

```diff
@@ -15,7 +16,9 @@ import java.util.*
 import javax.servlet.DispatcherType
 import javax.servlet.FilterRegistration
 
-data class Config(val database: Neo4jConfig): Configuration()
+typealias MultiMap<K,V> = Map<K, List<V>>
+
+data class Config(val database: Neo4jConfig, val siteTypeCategories: MultiMap<String, String>): Configuration()
```

Comment:
andrew and i discussed it and decided it would be better to keep the config out of code for now because we do expect the number of mappings to increase. config data should remain unchanged during the course of an app's runtime, it's configuration on how to run and unless specifically mutable, it should remain immutable. we are exposing it via the rest endpoint so the front-end doesn't have to keep track of the different mappings, they can just ask the backend and any new categories/subcategories will automatically show up in the UI.